### DISCUSSION TITLE
Co worker lost admin password

### DISCUSSION THREAD

**Discussion by AlanGreyjoy** [requarks/wiki#4146] [General]
We are using the DO droplet from the marketplace.

I have tried this,
Went to https://bcrypt-generator.com/ and generated a hash

Then I ran this
docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = 'HASH-PASSWORD' WHERE email = 'YOUR-EMAIL';"

Console says UPDATE 1

When I try to login, still bad user and pass.

We can't do email reset, because they did not update the SMTP creds for the new SendGrid auth.

Any help would be appreciated.

**Comment by AlanGreyjoy** [comment]
Bueller...Bueller...Bueller

**Reply by benjamincarmel** [reply]
This isn't likely to help you get support there's only one main developer of this project

**Reply by AlanGreyjoy** [reply]
Bueller
